app/init/launch.ts

```diff
@@ -168,7 +168,7 @@ const launchToHome = async (props: LaunchProps) => {
             break;
         }
         case Launch.Normal:
-            if (props.coldStart) {
+            if (props.coldStart || AppState.currentState === 'active') {
```

Comment:
When the app is closed and a new notification is received, the app is started on the background. If you open the app from the app icon in the home screen of the device, the flag for `coldStart` is false, failing to run app entry and connect the websocket(s). This fixes it as the app state changes to `active`

Reply:
The reason to have this check here was around only calling the appEntry the first time we run, since any other run will imply a websocket reconnect, and entry will be handled there.

This change leads me to think that everytime we open the app from the app icon, no matter if the app was already started or not (and therefore, whether the websocket is reconnecting or not) will run this.

I wonder if the `AppState.currentState === 'active'` should be more in the `coldStart` calculation (as in if the state is not active, it is not really a cold start). But 0/5.